### ISSUE TITLE
Swiper in lazyload mode inits only after screen resize in a Jquery Mobile app.

### ISSUE THREAD

**Issue by BeWilled** [nolimits4web/Swiper#1244]
Hello, I've put together this fiddle to show how the swiper widget only inits after the window is resized when used in conjuction with jquery mobile. 

http://jsfiddle.net/x07k9wk7/2/

1. Open the fiddle
2. Resize the browser window
3. Swiper will load

Adding the swiper initiatior in the jQuery.ready() event makes this work. But for specific porpuses I need the init script to be inline. 

Best regards,
BeWilled

**Comment by nolimits4web** [comment]
Say hello to "amazing" logic of jQuery Mobile :) This problem happens because on moment of initialization Swiper is hidden, because JQM's current page is also hidden on app load. 
So you have 3 options:
* Option 1. You need to use some of JQM's callbacks and init/update Swiper when page becomes visible. 
* Option 2. Add these options to Swiper: `observer: true, observeParents: true`. It will be automatically reinitialized, but there is another issue in Swiper with this. So you also need to get the latest dev version where it is fixed https://github.com/nolimits4web/Swiper#build
* Option 3. Change jQM to some modern and better mobile framework ;)

**Comment by nolimits4web** [comment]
Closing as no further discussion
